packages/groups/src/components/Form.jsx

```diff
@@ -117,14 +129,17 @@ const Form = () => {
   useEffect(() => {
     (async () => {
       dispatch({ type: 'status', payload: 'Loading' })
-      const { entityFactory, taskService } = await services()
+      const { entityFactory, taskService, queryService } = await services()
       dispatch({ type: 'taskService', payload: taskService })
       const members = await entityFactory({ resource: 'member' })
       const member = await members.get(user.iss)
       const groups = await entityFactory({ resource: 'group' })
+      const { profiles } = await queryService.recommend()
```

Comment:
This can throw an exception when the user making this call either has no profile or did not consent to sharing their data [0]

We should wrap this call in a `try/catch` to not break the app.

[0] https://github.com/kernel-community/services/blob/main/packages/query/src/services/query.js#L58

Reply:
done